hadoop-plugin/src/test/groovy/com/linkedin/gradle/hadoop/HadoopPluginTest.groovy

```diff
@@ -43,7 +43,9 @@ class HadoopPluginTest {
   public void testApplyHadoopPlugin() {
     plugin.apply(project);
 
-    // Check that the hadoopRuntime configuration was created successfully.
+    // Check that hadoop configurations were created successfully.
```

Comment:
Can you find a way to add a unit test that adds a provided dependency, then checks that the provided dependency is NOT in the compile, runtime, default and hadoopRuntime configurations, but IS in the testCompile configuration?